src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -946,6 +975,12 @@ protected override void BeginProcessing()
                     }
                 }
 
+                if (MyInvocation.BoundParameters.ContainsKey(nameof(StrictModeVersion)))
```

Comment:
How about using `if (_strictmodeversion is not null)`?

Reply:
Null is a valid value for _strictmodeversion.